cms/src/components/groups/GroupEditor.vue

```diff
@@ -403,12 +409,12 @@ const saveChanges = async () => {
                         Offline changes
                     </LBadge>
                     <LButton
-                        v-if="groups && groups.length > 0"
-                        variant="secondary"
+                        v-if="groups && groups.length > 0 && open"
+                        variant="tertiary"
+                        size="sm"
+                        :icon="DocumentDuplicateIcon"
```

Comment:
Tiny suggestion: Add `title="Duplicate"` to this so that when you move your mouse over it it shows what the button will do.